### PR TITLE
cleanup svelte-check --ignore for  data-graphics

### PR DESCRIPTION
**Pull request by bcolloran**
note: full text search shows that `<Density` and `<ECDF` does not show up any where in our codebase. They include both mistyped values as well as an imports that are unavailable in our repo, so I think they either haven't been used for a long time, or may never have actually been adopted. Deleted these under the "you ain't gonna need it" principle.

part of #3424

### CODE REVIEW

File: web-common/src/components/data-graphic/marks/Line.svelte
Comment (bcolloran) [inline web-common/src/components/data-graphic/marks/Line.svelte:65]: note: no need to pass `pathDefined = pathDoesNotDropToZero`, because `pathDoesNotDropToZero` is the dafault when this argument is omitted

File: web-common/src/components/data-graphic/marks/Density.svelte
Comment (bcolloran) [inline web-common/src/components/data-graphic/marks/Density.svelte:1]: note: full text search shows that `<Density` does not show up any where in our codebase. This file has both mistyped values as well as an import that is unavailable in our repo, so I think it hasn't been used for a long time, or it may never have actually been adopted. deleting it under the "you ain't gonna need it" principle

File: web-common/src/components/data-graphic/constants/types.d.ts
Comment (bcolloran) [inline web-common/src/components/data-graphic/constants/types.d.ts:3]: note that
(1) I could not find any instances of Date being used in this type (it always seems to be used in numeric operations, but please double check me on this)
(2) these fields are very commonly set to undefined by way of the `DEFAULT_COORDINATES` const which is used all over the place

```
export const DEFAULT_COORDINATES: DomainCoordinates = {
  x: undefined,
  y: undefined,
};
```

File: web-common/src/components/data-graphic/marks/Density.svelte
Comment (bcolloran) [inline web-common/src/components/data-graphic/marks/Density.svelte:1]: file never used

File: web-common/src/components/data-graphic/marks/ECDF.svelte
Comment (bcolloran) [inline web-common/src/components/data-graphic/marks/ECDF.svelte:1]: file never used

File: web-common/src/components/date-picker/Litepicker.svelte
Comment (bcolloran) [inline web-common/src/components/date-picker/Litepicker.svelte:267]: expanding these tailwind styles to clear a warning about use of  @apply

**Comment by bcolloran** [comment]
hey @djbarnwal , I have another review for you. I think it should all be pretty straightforward, but the one place where your expertise on this part of the codebase will be a big help is double checking whether "Date" is ever actually used in the DomainCoordinate type. Thanks!

File: web-common/src/components/date-picker/Litepicker.svelte
Comment (ericpgreen2) [reply web-common/src/components/date-picker/Litepicker.svelte:267]: I recommend `<style lang="postcss">`, which should remove these warnings.

**Review by ericpgreen2 — APPROVED** [review]
Looks good 💪. I only recommend making the `lang="postcss"` change before merging.

File: web-common/src/components/date-picker/Litepicker.svelte
Comment (bcolloran) [reply web-common/src/components/date-picker/Litepicker.svelte:267]: Oooh thanks, great tip!! Didn't know we could do that in svelte

**Pull request merged**
